examples/notebooks/Watch_Warning_and_Advisory_Plotting.ipynb

```diff
@@ -0,0 +1,728 @@
+    "\n",
+    "# Keep track of the earliest and latest reftime for title\n",
+    "# start with the first time from the first object in the response\n",
+    "time_str = str(response[0].getDataTime().getRefTime())\n",
+    "# truncate the decimal seconds for datetime parsing\n",
+    "time_str = time_str[:-4]\n",
+    "# convert to datetime object for easy comparison\n",
+    "first_time = datetime.strptime(time_str, '%Y-%m-%d %H:%M:%S')\n",
+    "last_time = datetime.strptime(time_str, '%Y-%m-%d %H:%M:%S')\n",
+    "\n",
+    "for ob in response:\n",
+    "    \n",
+    "    # get the geometry for the object\n",
+    "    poly = ob.getGeometry()\n",
+    "    # get the reftime for the object\n",
+    "    ref  = ob.getDataTime().getRefTime()\n",
+    "    \n",
+    "    # do not plot if phensig is blank (SPS)\n",
+    "    if ob.getString('phensig'):\n",
+    "        \n",
+    "        # look at the reftime\n",
+    "        # convert reftime to a string and parse the decimal seconds\n",
+    "        ref_str = str(ref)\n",
+    "        ref_str = ref_str[:-4]\n",
+    "        # convert reftime to a datetime object for comparison\n",
+    "        ref_time = datetime.strptime(ref_str, '%Y-%m-%d %H:%M:%S')\n",
+    "        # compare current time with first and last times and set if appropriate\n",
+    "        if ref_time is not None:\n",
+    "            if ref_time < first_time:\n",
+    "                first_time = ref_time\n",
+    "            elif ref_time > last_time:\n",
+    "                last_time = ref_time\n",
+    "        \n",
+    "        # get the phensig and sig values from object\n",
+    "        phensigString = ob.getString('phensig')\n",
+    "        sig = ob.getString('sig')\n",
+    "\n",
+    "        # set the geometries based on whether it's a MultiPolygon or Polygon\n",
+    "        if poly.geom_type == 'MultiPolygon':\n",
+    "            geometries = np.array([])\n",
+    "            geometries = np.append(geometries,MultiPolygon(poly))\n",
+    "            geom_count = \", \" + str(len(geometries)) +\" geometries\"\n",
+    "        else:\n",
+    "            geometries = np.array([])\n",
+    "            geometries = np.append(geometries,Polygon(poly))\n",
+    "            geom_count=\"\"\n",
+    "\n",
+    "        for geom in geometries:\n",
+    "            bounds = Polygon(geom)\n",
+    "            intersection = bounds.intersection\n",
+    "            geoms = (intersection(geom) for geom in geometries if bounds.intersects(geom))\n",
+    "    \n",
+    "        # Store the unique phensigs\n",
+    "        if not phensigString in phensigs:\n",
+    "            phensigs.append(phensigString)\n",
+    "            # Optional printout of unique Phensigs\n",
+    "            print(get_title(phensigString) \n",
+    "             + \" (\" + phensigString + \") issued at \" + str(ref)\n",
+    "             + \" (\"+str(poly.geom_type) + geom_count + \")\")\n",
```

Comment:
I'd be careful what you are printing out here. I would not include the issue time, geom type, or geom count. If you want to show that you can print this type of information out, I would do it outside of the loop since this is just grabbing the first phensig of each type.

Reply:
Yeah, it was honestly just there more as a sanity check.  Something similar was in the existing notebook, but commented out (http://unidata.github.io/python-awips/examples/generated/Watch_and_Warning_Polygons.html).
I could just comment it out, since I already have a comment above it saying it's an "optional printout"?

Reply:
even commented out, it can be confusing. It took me a second to figure out what it was doing (I thought were were missing plotting WWA's until I realized it was just getting the unique phensigs. I would just move it up out of this if statement and maybe comment it out?